resolvers/ambulance.js

```diff
@@ -5,30 +5,37 @@ const validators = require('../utils/validators');
 
 const ambulanceResolvers = {
   Query: {
-    ambulances: () =>
-      db.ambulance.findAll({
+    ambulances: () => {
+      validators.validateRole(['COMMANDER']);
```

Comment:
we want `validateRole` to take the user's current role, as it should not be accessible from within `validators` - this will require you to hard code a user role in every one of these resolvers, example below
```
...
// TODO: remove this after authentication is implemented
const userRole = 'COMMANDER';
...
validators.validateRole(userRole, ['COMMANDER']);
```